policies-procedures-and-docs/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Policies, Procedures & Documents
+image: https://images.unsplash.com/photo-1683178861337-ca70ef8c0db3
+photos:
+  name: Zetong Li
+  license: CC-BY
+  url: https://images.unsplash.com/photo-1683178861337-ca70ef8c0db3
+---
+
+<h1>{{ page.title }}</h1>
+
+<ul>
+  {% for doc in site.policies-procedures-and-docs %}
```

Comment:
So, apparently, Jekyll doesn't loop through nested directories to list out files. Right now, it only sees files if they are located at the root.

We have 3 options:
- **Break the strict categorisation** that exists now (i.e pull out the files from inside subfolders and place them at the root). Easy solution, but it messes with the organisation.
- **Use a `site.categories` object:** Each file is assigned a category in their frontmatter; we loop through using this and output the pages. (More complex and less sustainable, but it maintains the current structure.)
- Whatever other option I missed.

Reply:
Have you tried with `site.pages`?